app.js

```diff
@@ -3,6 +3,23 @@ var express = require('express');
 var path = require('path');
 var cookieParser = require('cookie-parser');
 var logger = require('morgan');
+var session = require('express-session');
+let randomString = require('./utils/utils');
+let { v4: uuidv4 } = require('uuid');
+var MongoDBStore = require('connect-mongodb-session')(session);
+
+const passport = require('passport');
+
+var store = new MongoDBStore({
+  uri: 'mongodb://localhost:27017/connect_mongodb_session_test',
```

Comment:
Let us connect to the same db, The session data will be saved to a different collection so it will not impact other data.

Reply:
done